downstream/modules/automation-hub/con-create-api-token.adoc

```diff
@@ -3,7 +3,7 @@
 
 Before you can interact with automation hub by uploading or downloading collections, you must create an API token. The automation hub API token authenticates your `ansible-galaxy` client to the Red Hat automation hub server.
```

Comment:
Should we use attribute for "automation hub"?

Reply:
Hey @EMcWhinn! Thanks for this note. For some reason, the attribute was not rendering correctly in this document. It was repeatedly capitalising 'Automation Hub', although our style guide has it as 'automation hub'. Neither Ian nor I could figure out why that was happening, but I decided just to spell out automation hub instead of using the attribute in these cases. Post-2.4, I'll revisit this, and if I can resolve the attribute capitalisation error, I'll change these to attributes.

Reply:
Hey @emurtoug, I think this happened to me recently as well now that you mention, so I had to spell it out too. All looks good so!